chord_metadata_service/metadata/settings.py

```diff
@@ -51,6 +51,15 @@
 
 CHORD_URL = os.environ.get("CHORD_URL")  # Leave None if not specified, for running in other contexts
 
+# SECURITY WARNING: Don't run with AUTHZ_ENABLED turned off in production,
+# unless an alternative permissions system is in place.
+#  - This needs to be here to avoid a circular import with settings.py
+BENTO_AUTHZ_ENABLED: bool = os.environ.get("BENTO_AUTHZ_ENABLED", "true").strip().lower() == "true"
+
+BENTO_AUTHZ_SERVICE_URL: str = (
+    os.environ.get("BENTO_AUTHZ_SERVICE_URL", "http://authz.local").strip().rstrip("/") if BENTO_AUTHZ_ENABLED else ""
+)
+
```

Comment:
Regarding the test errors when in dev mode.
I did a quick test and we can alter `BENTO_AUTHZ_SERVICE_URL` based on `sys.argv` to detect if we are running tests. The following code block allowed me to run all tests in DEV with no errors:
```python
if len(sys.argv) > 1 and sys.argv[1] == 'test':
    BENTO_AUTHZ_SERVICE_URL = "http://authz.local"
```

Reply:
done